packages/migrations/package.json

```diff
@@ -14,7 +14,7 @@
     "db:init": "pnpm db:create && pnpm migration:run",
     "db:migrator": "node --experimental-specifier-resolution=node --loader ts-node/esm src/index.ts",
     "migration:run": "pnpm db:migrator up",
-    "test": "node --experimental-specifier-resolution=node --loader ts-node/esm ./test/*.test.ts"
+    "test": "TS_NODE_TRANSPILE_ONLY=true node --experimental-specifier-resolution=node --loader ts-node/esm ./test/index.ts"
```

Comment:
I did that because we use `Storage` in tests and otherwise I would have to include it and a few more paths in `tsconfig`.
It's type-checked anyway by `pnpm typecheck`.